service-discovery/kubernetes/src/main/java/io/smallrye/stork/servicediscovery/kubernetes/KubernetesServiceDiscovery.java

```diff
@@ -44,23 +47,27 @@ public KubernetesServiceDiscovery(String serviceName, ServiceDiscoveryConfig con
         if (namespace != null && namespace.equalsIgnoreCase("all")) {
             allNamespaces = true;
         }
+        String refreshPeriod = config.parameters().get("refresh-period");
```

Comment:
can we read the `refreshPeriod` in the CachingServiceDiscovery class?

Reply:
Let me check

Reply:
done